### ISSUE TITLE
What is threshold exactly in meshing_merge_close_vertices?

### ISSUE THREAD

**Issue by JunzheJosephZhu** [cnr-isti-vclab/PyMeshLab#377]
Why does 1% correspond to 1/10000 of bounding box diagonal? Does that mean 100% would correspond to 1/100 of bounding box diagonal?

**Comment by alemuntoni** [comment]
Yep. I don't know why exactly 1/10000 was chosen as default value (I did not write the filter), but in any case it would be nonsense setting 100% to the bounding box diagonal.
